src/snapshots/crash_and_ellipsis_test.md

```diff
@@ -221,11 +221,13 @@ testEllipsis = |_| ...
 
 # Test crash statement
 testCrash : U64 -> U64
-testCrash = |_| "This is a crash message"
+testCrash = |_| 
+"This is a crash message"
 
 # Test crash with different message
 testCrashSimple : U64 -> U64
-testCrashSimple = |_| "oops"
+testCrashSimple = |_| 
+"oops"
```

Comment:
this looks like a formatting regression

Reply:
Same as above, this is caused by [this](https://github.com/JRI98/roc/blob/fe19ff4a44938df9f3efcb84ee6eacd0edfe8faa/src/snapshots/crash_and_ellipsis_test.md?plain=1#L187) malformed statement and is a separate issue where `crash` is not being kept in the formatted code. Will be fixed when that is actually kept in the output.
For reference, this is the original line:
```
testCrashSimple = |_| crash "oops"
```